cluster/common.sh

```diff
@@ -446,8 +446,19 @@ function yaml-quote {
 # Builds the RUNTIME_CONFIG var from other feature enable options (such as
 # features in alpha)
 function build-runtime-config() {
-  # There is nothing to do here for now. Just using this function as a placeholder.
-  :
+  # If a policy provider is specified, enable NetworkPolicy API.
+  if [[ -n "${NETWORK_POLICY_PROVIDER}" ]]; then
```

Comment:
AWS is what I noticed: https://storage.googleapis.com/kubernetes-jenkins/logs/kubernetes-e2e-aws/278/build-log.txt
```
/jenkins-master-data/jobs/kubernetes-e2e-aws/workspace/kubernetes/hack/e2e-internal/../../cluster/../cluster/aws/../../cluster/../cluster/../cluster/../cluster/aws/../../cluster/common.sh: line 479: NETWORK_POLICY_PROVIDER: unbound variable
```